infra.bs

```diff
@@ -239,6 +239,39 @@ paths would be an insurmountable task and not productive as they would not stand
 as well as the single algorithm would. Therefore performance is best left as a field to compete
 over.
 
+<h4 id=algorithm-conformance-limits>Avoid limits on algorithm inputs</h4>
```

Comment:
I think this should be a h3 since it's not really about the must/may/etc. stuff that the "Conformance" section is mostly about.